docs/request-audit-headers.md

```diff
@@ -25,28 +25,13 @@ For synchronous (i.e. non-bulk) requests, BFD users SHALL include meaningful val
 * `BlueButton-UserId`: the unique ID in the frontend system for the user (e.g. beneficiary) that data is being requested on behalf of.
 * `BlueButton-User`: the human-readable login ID (e.g. email address) in the frontend system for the user (e.g. beneficiary) that the data is being requested on behalf of.
 * `BlueButton-BeneficiaryId`: the unique beneificary ID (e.g. `Patient.id`) from BFD for the user the beneficiary whose data is being requested.
+* `x-forwarded-for`: a standard header for identifying the originating IP address of a client connecting to a web server through an HTTP proxy or a load balancer.
 
 ## Asynchronous Requests
 
 For asynchronous (i.e. bulk) requests, BFD users SHALL include meaningful values for as many as possible of the following HTTP headers.
 
-### BCDA
```

Comment:
Will requests that contain these deprecated headers be rejected?
For reference, the code in BCDA which sets these headers is [here](https://github.com/CMSgov/bcda-app/blob/master/bcda/client/bluebutton.go#L286-L310).
CC: @mtrang1263

Reply:
No. These headers are only used for auditing purposes.